Package.swift

```diff
@@ -24,8 +24,8 @@ let package = Package(
     dependencies: [
         .package(name: "MapboxDirections", url: "https://github.com/mapbox/mapbox-directions-swift.git", from: "2.4.0"),
         .package(name: "MapboxMobileEvents", url: "https://github.com/mapbox/mapbox-events-ios.git", from: "1.0.0"),
-        .package(name: "MapboxNavigationNative", url: "https://github.com/mapbox/mapbox-navigation-native-ios.git", from: "98.0.0"),
-        .package(name: "MapboxMaps", url: "https://github.com/mapbox/mapbox-maps-ios.git", .exact("10.5.0-rc.1")),
+        .package(name: "MapboxNavigationNative", url: "https://github.com/mapbox/mapbox-navigation-native-ios.git", from: "99.0.0"),
+        .package(name: "MapboxMaps", url: "https://github.com/mapbox/mapbox-maps-ios.git", .exact("10.5.0")),
```

Comment:
This should be a `from:` dependency, now that MapboxMaps is on a final release. If we don’t depend on any new APIs in v10.5.0 specifically, then roll the minimum requirement back to v10.4.1, which was introduced in #3806.

Reply:
Ah will fix this now

Reply:
When setting `MapView.location.options.puckType` to `nil` behavior would be different on `v10.4.1` and `v10.5.0`. If we keep depending on `v10.4.1` as a minimum version we'll have to add check for both versions. I think it'd be better to depend on `v10.5.0` from now on.